ocs_ci/deployment/deployment.py

```diff
@@ -579,6 +580,21 @@ def deploy_ocs_via_operator(self, image=None):
         deviceset_data = cluster_data["spec"]["storageDeviceSets"][0]
         device_size = int(config.ENV_DATA.get("device_size", defaults.DEVICE_SIZE))
 
+        logger.info(
+            "Flexible scaling is available from version 4.7 on LSO cluster with less than 3 zones"
+        )
+        ocs_version = float(config.ENV_DATA["ocs_version"])
+        zone_num = get_az_count()
+        if (
+            config.DEPLOYMENT.get("local_storage")
+            and ocs_version >= 4.7
+            and zone_num < 3
+        ):
```

Comment:
We should be using semantic_version according to [Petr's email](https://mailman-int.corp.redhat.com/archives/ocs-qe/2021-March/msg00433.html) a couple weeks back. This will basically prevent comparisons like this from breaking once ocs_version reaches something like 4.10 or 4.11. This code expects an ocs_version of 4.10 to be greater than 4.7 but the float 4.10 is less than 4.7. By using semantic_version, we can avoid this.

The email has a small example of how to import and use semantic_version but let me know if it's still unclear.